src/static/bitbucket/index.js

```diff
@@ -0,0 +1,155 @@
+const endpoint = window.location.origin;
```

Comment:
This is not great obviously.
I wanted to avoid adding a) Something heavy like React and b) a bundler like webpack as I wanted this to be simple and static...

I think it's probably worth adding one or both though. Thoughts?